e2e/k8s/k8s.go

```diff
@@ -0,0 +1,20 @@
+package k8s
+
+import (
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+)
+
+func Client() (*kubernetes.Clientset, error) {
+	// creates the in-cluster config
```

Comment:
Comments nitpick but I'd like these to be sentences with a capital and full stop.

Reply:
Sure, done :)